bin/kyuubi-logo

```diff
@@ -15,18 +15,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
```

Comment:
We'd be better to keep this blank line, separating the license block and the single line comment below.

Reply:
Shall we bring back this empty line? And then this PR is ready for merge.